packages/jest-preset-default/scripts/travis-fold-passes-reporter.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+const { DefaultReporter } = require( '@jest/reporters' );
+
+class TravisFoldPassesReporter extends DefaultReporter {
+	constructor( ...args ) {
+		super( ...args );
+		this.foldedTestResults = [];
+	}
+
+	flushFoldedTestResults() {
+		if ( this.foldedTestResults.length > 0 ) {
+			this.log( 'travis_fold:start:TravisFoldPassesReporter' );
+			this.log(
+				`...${ this.foldedTestResults.length } passing test${
+					this.foldedTestResults.length === 1 ? '' : 's'
+				}.`
+			);
+			this.foldedTestResults.forEach( ( args ) => super.onTestResult( ...args ) );
+			this.log( 'travis_fold:end:TravisFoldPassesReporter' );
+			this.foldedTestResults = [];
+		}
+	}
+
+	onTestResult( ...args ) {
+		if ( args[ 1 ].numFailingTests === 0 && ! args[ 1 ].failureMessage ) {
```

Comment:
Purely from a readability perspective, it might help to assign `args[ 1 ]` to a named variable. Even at this moment, I have no idea what `args[ 1 ]` is supposed to represent.

Reply:
Good call.